content/post/000-ipfs-in-web-browsers.md

```diff
@@ -0,0 +1,74 @@
+
+Decentralization feels slow in providing user-centric approaches to the internet's biggest problems. Right now true user agency is often too technical, too fragile and too time-consuming to be the default choice.
+
+But we're making headway. And we'd like to share some collaborations the IPFS project has had in the works for a while, which bring us just a few steps closer to making unmediated access to information *just work*... by solving that "last mile" problem and *integrating IPFS directly into web browsers*.
+
+## Stages
+
+The path to a truly decentralized web is a long one. For over 30 years the browser has been a *client* - but a foundational concept in P2P systems is that a participant is both a client *and a server*. Web browser vendors and web standards bodies have not designed for this architectural shift, so we're breaking it down into steps.
+
+Initially we're aiming to ease access to the IPFS HTTP gateways - creating a bridge between the billions of people on the centralized web and content on the distributed web. We expect some experimentation in the next phase - adjusting node behavior based on browser APIs, device capabilities, bandwidth availability and power consumption - growing adoption with our browser partners as we learn, and building common understanding at standards bodies. Finally we aim to have multiple browsers implementing IPFS natively and interoperably.
```

Comment:
Maybe this needs a paragraph before this about where we are? I think this is picking up from "current day" where we *already* have the IPFS HTTP gateway that is bridging access to the dweb and saying we will make it even more seamless to use this resource - but I think it's worth reflecting first on all we've already done to bridge access to the dweb before explaining our stages going forward...

Reply:
I might also flip the stages diagram ahead of this, so that users have something with clear steps in their head before reading this paragraph, which doesn't clearly delineate where one stage begins and another ends...

Reply:
```suggestion
Initially, we're aiming to ease access to the IPFS HTTP gateways - creating a bridge between the billions of people on the centralized web and content on the distributed web. We expect some experimentation in the next phase - adjusting node behavior based on browser APIs, device capabilities, bandwidth availability and power consumption - while we grow adoption with our browser partners as we learn and build common understanding at standards bodies. Ultimately, we aim to have multiple browsers implement fully native (and interoperable!) IPFS support.
```

Reply:
Suggest also adding another paragraph below to lead into the 3 sections on browser updates. Something along the lines of "As part of this progression to native IPFS support, we've had ongoing collaborations with Firefox, Brave, Opera, and other browsers. Read on to hear the latest status and our progress so far!"

Reply:
fixed and fixed!